src/core/time.jl

```diff
@@ -20,16 +20,6 @@ function gettime(ts::VariableTimestep)
 	return ts.current
 end
 
-"""
-	is_time(ts::AbstractTimestep, t::Int)
-
-Return true or false, true if the current time (year) for `ts` is `t`
-"""
-function is_time(ts::AbstractTimestep, t::Int) 
-	Base.depwarn("`is_time(ts, t)` is deprecated. Use comparison operators with TimestepValue objects instead: `ts == TimestepValue(t)` \n$(stacktrace())", :is_time)
-	return gettime(ts) == t
-end
-
```

Comment:
same question for the rest of these as well: do we just want to delete the old functionality entirely, or should we just turn it into helpful error messages?

Reply:
Agreed I’m not entirely sure what the intention was.  It’s easy for me to go back and chang warnings to errors, I’ll post on Slack now.

Reply:
@ckingdon95 for these I could add a helpful error message, but we probably still don't want to export these in the main API right? That said if I don't export them, users tried to use them without `Mimi.` prefix, then they're going to get an error anyways.